README.md

```diff
@@ -4,11 +4,11 @@
 
 1. You will need the Java JDK 1.8+ installed, downloadable from the [Java SDK Downloads](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html) page.
 
-2. You will need Maven 3.+ installed.  Linux users can use their package manager to install at the command line (example for apt package management`sudo apt-get install maven`), and similarly OSX users using brew can `brew install maven`.  Windows users can install via [Chocolatey](https://chocolatey.org/) or by downloading the installer at the [Maven downloads](http://maven.apache.org/download.cgi_) page.
+2. You will need Maven 3.+ installed.  Linux users can use their package manager to install at the command line (example for apt package management`sudo apt-get install maven`), and similarly OSX users using brew can `brew install maven`.  Windows users can install via [Chocolatey](https://chocolatey.org/) (`choco install maven`) or by downloading the installer at the [Maven downloads](http://maven.apache.org/download.cgi_) page.
```

Comment:
I would just use this as the direct download link, as it doesn't have the mirror stuff:
https://maven.apache.org/download.cgi